docs/integrations/integrations.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+# OpenCost Integrations
+
+OpenCost may be integrated with a wide variety of technologies. Below is a list of known OpenCost integrations.
+
+## Cloud Integrations
+* [Azure](http://aka.ms/aks/OpenCost-AKS)
```

Comment:
Not sure if you'd want to link to the Azure page instead maybe?

```suggestion
* [Azure](../configuration/azure-prices)
```

Reply:
I thought the Azure blog post had a lot of other context, but it makes sense to keep things internal when they're available.